From 51760b0a3e6cdb0a46186a9e4e6117e6625f56e5 Mon Sep 17 00:00:00 2001
From: Victor Wan <victor.wan@amlogic.com>
Date: Mon, 24 Mar 2014 11:33:37 +0800
Subject: [PATCH 3794/5965] meson: config: change config name

meson8_vlsi_defconfig for Android/busybox on M8 emulator
meson8b_vlsi_defconfig for Android/busybox on M8baby emulator
---
 arch/arm/configs/meson6_defconfig                         | 0
 arch/arm/configs/meson6tv_defconfig                       | 0
 arch/arm/configs/meson6tvd_defconfig                      | 0
 arch/arm/configs/meson8_defconfig                         | 0
 arch/arm/configs/{meson8_vlsi => meson8_vlsi_defconfig}   | 0
 arch/arm/configs/{meson8b_vlsi => meson8b_vlsi_defconfig} | 0
 6 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 arch/arm/configs/meson6_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson6tv_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson6tvd_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson8_defconfig
 rename arch/arm/configs/{meson8_vlsi => meson8_vlsi_defconfig} (100%)
 rename arch/arm/configs/{meson8b_vlsi => meson8b_vlsi_defconfig} (100%)
 mode change 100755 => 100644

diff --git a/arch/arm/configs/meson6_defconfig b/arch/arm/configs/meson6_defconfig
old mode 100755
new mode 100644
diff --git a/arch/arm/configs/meson6tv_defconfig b/arch/arm/configs/meson6tv_defconfig
old mode 100755
new mode 100644
diff --git a/arch/arm/configs/meson6tvd_defconfig b/arch/arm/configs/meson6tvd_defconfig
old mode 100755
new mode 100644
diff --git a/arch/arm/configs/meson8_defconfig b/arch/arm/configs/meson8_defconfig
old mode 100755
new mode 100644
diff --git a/arch/arm/configs/meson8_vlsi b/arch/arm/configs/meson8_vlsi_defconfig
similarity index 100%
rename from arch/arm/configs/meson8_vlsi
rename to arch/arm/configs/meson8_vlsi_defconfig
diff --git a/arch/arm/configs/meson8b_vlsi b/arch/arm/configs/meson8b_vlsi_defconfig
old mode 100755
new mode 100644
similarity index 100%
rename from arch/arm/configs/meson8b_vlsi
rename to arch/arm/configs/meson8b_vlsi_defconfig
-- 
2.19.0

